jet/cdc/src/main/java/com/hazelcast/samples/jet/cdc/Order.java

```diff
@@ -0,0 +1,17 @@
+package com.hazelcast.samples.jet.cdc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public record Order (
+        int id,
+        @JsonProperty("order_date")
+        Date orderDate,
+        int purchaser,
+        int quantity,
+        @JsonProperty("product_id")
+        int productId
+) implements Serializable {
```

Comment:
Is is possible to use compact for order instead of Serializable?

Reply:
yes, in fact I directly enable compact at the beginning. I will drop implements Serializable, so it's not confusing